src/main/java/com/checkout/payments/CardSource.java

```diff
@@ -14,7 +15,7 @@
 @AllArgsConstructor
 public class CardSource implements RequestSource {
 
```

Comment:
If `@Override public String getType()` is never used, I believe we can remove it from the interface. We can still use the interface (marker interface) for polymorphism if strictly necessary, or just deprecate it to be removed in a future version.